src/applications/vaos/tests/appointment-list/components/RequestedAppointmentDetailsPage.unit.spec.js

```diff
@@ -20,6 +20,7 @@ import {
   renderWithStoreAndRouter,
 } from '../../mocks/setup';
 
+// TODO: Delete this test suite since these test cases are now covered by VARequestLayout.unit.spec
```

Comment:
Ditto.

Reply:
This is an old comment that I forgot to delete.